linux-requirements.txt

```diff
@@ -1 +1,8 @@
 pygobject~=3.22.0
```

Comment:
Yes. Then install instructions of different OSes will need to be created in readme.